cloudevents/README.md

```diff
@@ -0,0 +1,111 @@
+# Debezium CloudEvents Demo
+
+This demo automatically deploys the topology of services as defined in the [Debezium Tutorial](http://debezium.io/docs/tutorial/).
+
+## Preparations
+
+```shell
```

Comment:
Should we add a step to run `mvn install` for avro-data-converter?

Reply:
Yes, good point.